src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -57,10 +90,22 @@ const ExpandedNode: FC<ExpandedNodeProps> = ({
             onChange={(e): void => setComments(e.target.value)}
           ></textarea>
           <div className={styles.footer}>
-            <ConfirmedActionButton type="accept" size="large" />
+            <ConfirmedActionButton
+              type="accept"
+              size="large"
+              onConfirm={(): void => {
+                onConfirm('Accepted', pathwayState.action);
+              }}
+            />
           </div>
           <div className={styles.footer}>
-            <ConfirmedActionButton type="decline" size="large" />
+            <ConfirmedActionButton
+              type="decline"
+              size="large"
+              onConfirm={(): void => {
+                onConfirm('Declined', []);
```

Comment:
Agreed, I feel like when I tried this I was getting Typescript errors.  I made the change.  Do you know why on line 54 Typescript  doesn't like `action?.length > 0`... I had to make it `action && action.length > 0`

Reply:
Seems upset because you can't compare use compare 0 with undefined using `>`.

Found pretty much the exact issue here: https://github.com/microsoft/TypeScript/issues/34593, but I like your solution better than the workaround proposed in the issue.